cds_ils/importer/serializers/schema.py

```diff
@@ -121,10 +121,17 @@ def records_statuses(self, data, **kwargs):
         """Return correct record statuses."""
         children_entries_query = ImportRecordLog.query \
             .filter_by(import_id=data.get('id'))
+
+        first_entry = children_entries_query.first()
+
+        initial_id = first_entry.id if bool(first_entry) else 0
```

Comment:
the thing is to check if first_entry exists before checking if it has an id otherwise it will give an error!

Reply:
Should be the same from what i can read. In both cases it will raise an error because the NoneType has no "id" attribute in case of the first record not being imported yet

Reply:
then it could be simply
```suggestion
        initial_id = 0
        if first_entry:
            initial_id = first_entry.id
```
I find it strange to cast the object to bool just to achieve one line, in 3 months I would be confused why the type cast is there